esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -372,23 +372,20 @@ define([
                 DropZoneFolders: true
             });
 
+            this.filter.setValue(this.id + "MaxNumberOfFiles", 1000);
+            this.filter.emit("apply");
             this.initWorkunitsGrid();
 
             this.filter.on("clear", function (evt) {
                 context.refreshGrid();
             });
             this.filter.on("apply", function (evt) {
-                context.refreshGrid();
-            });
-            topic.subscribe("hpcc/dfu_wu_completed", function (topic) {
-                context.refreshGrid();
-            });
-
-            WsDfu.DFUQuery({
-                request: {}
-            }).then(function (response) {
-                if (lang.exists("DFUQueryResponse.Warning", response)) {
-                    if (response.DFUQueryResponse.Warning) {
+                var retVal = context.filter.toObject();
+                WsDfu.DFUQuery({
+                    request: {}
+                }).then(function (response) {
+                if (lang.exists("DFUQueryResponse.DFULogicalFiles.DFULogicalFile", response)) {
+                    if (response.DFUQueryResponse.NumFiles > 100000 && !retVal.MaxNumberOfFiles) {
```

Comment:
Workflow goes as such. 
1. User comes in filter is set to 1000 files no matter what
2. If user clears and submits filter for example we need to know if the number of files is greater than 100,000 (in the response) and if also if they removed that MaxNumberOfFiles if so,
3. Show Toaster message.